winbuild/build_prepare.py

```diff
@@ -262,7 +284,7 @@ def cmd_msbuild(
                 "<UserDefines></UserDefines>": "<UserDefines>FT_CONFIG_OPTION_SYSTEM_ZLIB;FT_CONFIG_OPTION_USE_PNG;FT_CONFIG_OPTION_USE_HARFBUZZ;FT_CONFIG_OPTION_USE_BROTLI</UserDefines>",  # noqa: E501
                 "<UserIncludeDirectories></UserIncludeDirectories>": r"<UserIncludeDirectories>{dir_harfbuzz}\src;{inc_dir}</UserIncludeDirectories>",  # noqa: E501
```

Comment:
Isn't `inc_dir` out of scope here now?

Reply:
No, this gets expanded in `build_dep` here: https://github.com/python-pillow/Pillow/blob/ceb0c575e8c61a2d2b164f7f3e06d867804e304e/winbuild/build_prepare.py#L568

Reply:
Ah, I see, thanks